setup.py

```diff
@@ -118,7 +118,56 @@ def finalize_options(self):
 
 EXTENSIONS = [
```

Comment:
Minor and not important:

I was trying to make sure all the modules here are included but they're not sorted like the files. If you're doing any of the changes then I did so here.

```python
   Extension(
        "ConfigSpace.hyperparameters.beta_float",
        sources=["ConfigSpace/hyperparameters/beta_float.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.beta_integer",
        sources=["ConfigSpace/hyperparameters/beta_integer.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.categorical",
        sources=["ConfigSpace/hyperparameters/categorical.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.constant",
        sources=["ConfigSpace/hyperparameters/constant.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.float_hyperparameter",
        sources=["ConfigSpace/hyperparameters/float_hyperparameter.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.hyperparameter",
        sources=["ConfigSpace/hyperparameters/hyperparameter.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.integer_hyperparameter",
        sources=["ConfigSpace/hyperparameters/integer_hyperparameter.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.normal_float",
        sources=["ConfigSpace/hyperparameters/normal_float.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.normal_integer",
        sources=["ConfigSpace/hyperparameters/normal_integer.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.numerical",
        sources=["ConfigSpace/hyperparameters/numerical.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.ordinal",
        sources=["ConfigSpace/hyperparameters/ordinal.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.uniform_float",
        sources=["ConfigSpace/hyperparameters/uniform_float.pyx"]
    ),
    Extension(
        "ConfigSpace.hyperparameters.uniform_integer",
        sources=["ConfigSpace/hyperparameters/uniform_integer.pyx"]
    ),
```

Reply:
Ahh I now see there is some ordering there already, I was just trying to do so based of the directory structure

Reply:
I added them in the order that I extracted the classes. I will change to your order as that makes more sense.